graphql/models/Transaction.graphql

```diff
@@ -32,13 +32,15 @@ enum TransactionType {
     BRANCH_FUND_DISBURSEMENT
     WALLET_PAYMENT
     WALLET_WITHDRAWAL
+    COMPANY_WITHDRAWAL
```

Comment:
@Oluwablin @davidubanyi  

What is the difference between a COMPANY_WITHDRAWAL and a BRANCH_FUND_DISBURSEMENT?